### PR TITLE
Revamp About and Community Standards

### PR DESCRIPTION
**Pull request by joshuaboniface**
1. Add a layout conducive to additional contact info in the Core Team section of the About page, and add my own additional contact info (others can add theirs later if they choose).
2. Revamp the wording of several sections of the community standards and reorganize for clarity.

### CODE REVIEW

File: docs/general/community-standards.md [hunk 1]
```diff
@@ -5,48 +5,53 @@ title: Community Standards
...
+
+Questions or comments regarding these standards should be forwarded to the [Project Leader](https://matrix.to/#/@joshuaboniface:bonifacelabs.ca) or [Core Team](xref:about).
 
 ## Changelog
 
 This document represents official Jellyfin project policy. Any changes to this document require a changelog entry here and approval by a Project Leader.
 
-2020-09-14, Joshua Boniface: Initial version of the community standards document. Based _very_ loosely on several CoCs including the Contributor Covenant, and various Forum rules I've read and written over the years.
+2020-09-14, Joshua Boniface: Initial version of the community standards document. Based *very* loosely on several CoCs including the Contributor Covenant, and various Forum rules I've read and written over the years.
+2022-09-03, Joshua Boniface: Update header and footer sections; reduce redundant wording; make rules clearer.
```
Comment (nielsvanvelzen) [inline docs/general/community-standards.md:57]: Did you check if this shows correctly in the site? In the GitHub markdown preview it doesn't have a line break.

Comment (thornbill) [reply docs/general/community-standards.md:57]: I just checked and both lines are combined as a single paragraph.

Comment (joshuaboniface) [reply docs/general/community-standards.md:57]: Interesting when I did my local docfx build it looked right. I'll add a `<br/>`.

Comment (joshuaboniface) [reply docs/general/community-standards.md]: Actually, used bullet points instead.

Comment (thornbill) [reply docs/general/community-standards.md]: Probably a difference in how docfx and docusaurus handle markdown.

File: docs/general/community-standards.md [hunk 2]
```diff
@@ -5,48 +5,53 @@ title: Community Standards
...
 
-- Jellyfin is a media server system for your own media collection. Our communities are not a place to obtain media. **Do not engage in, encourage, or facilitate piracy**. Do not ask questions about where to obtain media. Do not ask for access to other users' servers, or sell or transfer access to private servers in our communities.
+- If you have questions to ask of the community, **please choose the appropriate location** (see our [Getting Help](xref:getting-help) page) and **ask your question in full, immediately, with as much detail as possible**. Author issues aside, [Asking Questions the Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html) is a valuable resource and is recommended reading before engaging with any free and open source community, including ours. **Do not pester or harass** community members for help or to answer questions. **Do not require others to pry information** out of you. **Do not spam questions**; they will be answered when they are answered.
```
Comment (thornbill) [inline docs/general/community-standards.md:32]:
```suggestion
- If you have questions to ask of the community, **please choose the appropriate location** (see our [Getting Help](/docs/general/getting-help) page) and **ask your question in full, immediately, with as much detail as possible**. Author issues aside, [Asking Questions the Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html) is a valuable resource and is recommended reading before engaging with any free and open source community, including ours. **Do not pester or harass** community members for help or to answer questions. **Do not require others to pry information** out of you. **Do not spam questions**; they will be answered when they are answered.
```

File: docs/general/community-standards.md [hunk 3]
```diff
@@ -5,48 +5,53 @@ title: Community Standards
...
 
-- If the issue cannot be resolved between the contributors, any complaints, instances of explicit rulebreaking, or unresolvable disagreements with other community members may be directed towards the Administrative team, or the Project Leaders directly. You may do so through email (`team [at] jellyfin.org`) or on Matrix via a direct message. Please include details and context as appropriate.
+- If the issue cannot be resolved between the contributors, any complaints, instances of explicit rulebreaking, or unresolvable disagreements with other community members may be directed towards the Core Team or the Project Leader directly. You may do so through email (`team [at] jellyfin.org`) or on Matrix via a direct message. Please include details and context as appropriate. See the [about page](xref:about) for the list of Core Team members and ways to contact them.
```
Comment (thornbill) [inline docs/general/community-standards.md:40]:
```suggestion
- If the issue cannot be resolved between the contributors, any complaints, instances of explicit rulebreaking, or unresolvable disagreements with other community members may be directed towards the Core Team or the Project Leader directly. You may do so through email (`team [at] jellyfin.org`) or on Matrix via a direct message. Please include details and context as appropriate. See the [about page](/docs/general/about) for the list of Core Team members and ways to contact them.
```

File: docs/general/community-standards.md [hunk 4]
```diff
@@ -5,48 +5,53 @@ title: Community Standards
...
 - The team will review the complaint and decide on an action, including but not limited to: informal private guidance, informal private warnings, formal public warnings, temporary ban(s) from the various platforms, or permanent ban(s).
 
 - Formal warnings will be made under a "second-chance-only" policy. Once warned, if one repeats the same behaviour, the response will escalate as appropriate to the infraction.
 
-- Moderation tasks in some locations are delegated to other team members. This dispute resolution procedure applies anywhere under the Jellyfin umbrella. The Administrative team and Project Leaders retain final say in any dispute resolutions.
+- Moderation tasks in some locations are delegated to other, non-Core team members. This dispute resolution procedure applies anywhere under the Jellyfin umbrella. The Core team and Project Leader retain final say in any dispute resolutions.
+
+## Questions or Comments
+
+Questions or comments regarding these standards should be forwarded to the [Project Leader](https://matrix.to/#/@joshuaboniface:bonifacelabs.ca) or [Core Team](xref:about).
```
Comment (thornbill) [inline docs/general/community-standards.md:50]:
```suggestion
Questions or comments regarding these standards should be forwarded to the [Project Leader](https://matrix.to/#/@joshuaboniface:bonifacelabs.ca) or [Core Team](/docs/general/about).
```

**Review by thornbill — CHANGES_REQUESTED** [review]
The `xref:` style links do not work in docusaurus.

**Review by thornbill — APPROVED** [review]

**Pull request merged**